Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -37,6 +43,10 @@ @interface CDTURLSessionTask ()
  */
 @property (nonatomic, strong) NSArray *requestInterceptors;
 
+@property (nonnull, nonatomic, strong) NSArray *responseInterceptors;
+
+@property int remainingRetries;
```

Comment:
I don't think there is any point. You don't increment or decrement it atomically, so I don't think there's any protection here as you'll overwrite values if we're using several threads.